docs/UserGuide.md

```diff
@@ -334,25 +340,27 @@ Examples :
 --------------------------------------------------------------------------------------------------------------------
 
 ## Customisation
+*HelloFile* supports user customisation such as themes.
+You can choose a theme from our theme library, so that you can personalise *HelloFile* to your liking.
 
 ### Changing themes
-HelloFile comes in light and dark themes. To change the theme, navigate to the top left-hand corner of the application,
+HelloFile comes with many themes. To change the theme, navigate to the top left-hand corner of the application,
 and select `view`, followed by `theme`. A new pop up window will be shown with available themes along with a quick preview
- as illustrated below.
+as illustrated below.
 ![Fig. 1](images/screenshots/changing_themes.png)
 
 ![Fig. 2](images/screenshots/changing_themes2.png)
 
 ### Changing view sizes
 HelloFile allows you to resize the view of each sub components for a clearer view of tags and folders. You can simply
- drag each view to the size desired.
+drag each view to the size desired.
 ![Fig. 3. Resizing taskbar](images/screenshots/taskbar_resize.png)
 
 ![Fig. 4. Resizing filebar](images/screenshots/filebar_resize.png)
 
 ### Persistent customisation
-All of your customisation options are saved and persists even when you close the application. HelloFile remembers your choices
- so you can spend your time on more productive pursuits.
+All of your customisation options are saved and persist even when you close the application. HelloFile remembers your choices,
+so you can spend your time on more productive pursuits.
```

Comment:
Actually I think it's not all saved, only the window size and the theme is saved, the panel sizes are going to be reset to default every time.

Reply:
Ok